staff/views/singletournaments.py

```diff
@@ -261,11 +250,11 @@ def advance(request, pk):
         matches = currentround.matches.all()
         for i in matches:
             if i.winner is None:
-                if mikes_super_function(tournament.id, currentround.id, nextround.id) is False:
```

Comment:
Going to assume you've tested these changes you've made

Reply:
This hasn't been tested by hand, however the extra variables are never used in the function